docs/ref/clvm.md

```diff
@@ -323,34 +323,32 @@ The arithmetic operators `+`, `-`, `*`, `/` and `divmod` treat their arguments a
 ### Rounding
 
 ```chialisp
-brun '(/ (q . 1)  (q . 2))' => ()
-brun '(/ (q . 2)  (q . 2))' => 1
-brun '(/ (q . 4)  (q . 2))' => 2
+(/ 1  2) => ()
```

Comment:
This will confuse fewer people if you change

`brun '(/ (q . 1) (q . 2))'` into `run (/ 1 2)`

Reply:
yeah, we do mix up "clvm" and "chialisp". I'm a bit disappointed that we don't really have a clear reference of all the operators we support. I'm thinking that this should be it. However, for most users, showing examples in "clvm" isn't helpful, because that's not what you would write in. In a way, I think the low level aspects of CLVM should be documented as a "curiosity" somewhere, but for most humans, it's not really important. In all of these examples, we're documenting the behavior of the operators. quoting the operands is not relevant, and I don't think we should dilute the example with that.

Reply:
also, this is "chialisp" syntax, not "shell"